apps/web/tsconfig.json

```diff
@@ -21,5 +21,4 @@
       "./node_modules/@types"
     ]
   },
-  "include": ["./src/**/*.ts", "./src/**/*.vue"]
-}
+  "include": ["./src/**/*.ts", "./src/**/*.vue", "src/main.ts", "src/composables/router.ts", "src/composables/router.ts", "src/main.ts"]}
```

Comment:
Change to `"include": ["./src/**/*.ts", "./src/**/*.vue"]`.

Reply:
done